metecho/api/model_mixins.py

```diff
@@ -153,18 +154,18 @@ def queue_create_pr(
             additional_changes=additional_changes,
             issues=issues,
             notes=notes,
+            alert_assigned_dev=alert_assigned_dev,
             originating_user_id=originating_user_id,
         )
 
-    def finalize_create_pr(self, *, error=None, originating_user_id):
+    def finalize_create_pr(
+        self, *, alert_assigned_dev=False, error=None, originating_user_id
+    ):
         self.currently_creating_pr = False
         self.save()
 
-        # If appropriate:
-        # Get assigned_qa or assigned_dev
-        # Find user in DB if present
-        # Send email
-        self.try_to_notify_assigned_user()
+        if alert_assigned_dev:
+            self.try_to_notify_assigned_user()
```

Comment:
@wlonk If I'm following this right, we should define this `try_to_notify_assigned_user()` function to send in the email to the `assigned_dev` ?  I'm looking at [this line in `alert-assigned-role`](https://github.com/oddbird/Metecho/pull/310/commits/74d8bc87a5112a30a7437e89d2a54c410eeef184#diff-b5db231bd778f94eb85ba5aefdfcbb0eL383) as a reference.

Reply:
Ah, per @jgerigmeyer [here](https://oddbird.slack.com/archives/CH6EN9NMA/p1593456286001200), it's `assigned_qa`. Maybe we can re-verify that?

Reply:
Gotcha, more specifically, should the code related to sending the email be defined here?

Reply:
So it's not in this mixin because it only happens for `Task`, so we need the class that it's mixed-in-to to implement what its notification logic is—you'll note one implementation of this is just `pass` because we don't have anything for it to do, yet. In `Task`, though, it uses `User.notify`, which sends an email for now, but is likely to eventually grow other notification capacities.

Reply:
I hope that addresses your question, and I hope it's as-intended, but I'm not sure of either!

Reply:
OK, that makes sense. Thank you much !